src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -925,8 +925,10 @@ private void IsSelectedChanged(object sender, PropertyChangedEventArgs e)
         /// <param name="subset">Subset of nodes that should be used for the search instead of the complete set of nodes. This is a list of NodeSearchElement types</param>   
         internal IEnumerable<NodeSearchElementViewModel> Search(string search, IEnumerable<NodeSearchElement> subset = null)
         {
-            var foundNodes = Model.Search(search, 0, subset);
-            return foundNodes.Select(MakeNodeSearchElementVM);
+            //Search executed using Lucene.NET
+            var foundNodes = Search(search, true);
+            var foundNodesModels = foundNodes.Select(x => x.Model).Distinct();
```

Comment:
Can you elaborate why Distinct() was added? Were we getting duplicates before?

Reply:
I think there is no need to use Distinct() due that all the nodes returned by the Search should be different, then I've removed the Distinct() in the next sentence: 
https://github.com/DynamoDS/Dynamo/pull/14169/commits/8cc66441bc72272c560fc5f4200256e3804eb255